### PR TITLE
Fei Docs

### PR DESCRIPTION
**Pull request by thomas-waite**
# Summary
Adds Fei Docs. Based on Github pages and created from the `/docs` directory.

Expected that in the future that the docs are updated simultaneously when new features are added/changed.

### CODE REVIEW

File: docs/protocol/Security.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Protocol Security
+
+Fei Protocol needs to protect against malicious actors and attack that come in multiple categories. We attempt to firewall assets from each other as best as possible, so that exploits in one area don't affect others. We also use a variety of features to pre-emptively and retractively secure assets, such as the PCV Guardian and PCV Sentinel.
```
Comment (ElliotFriedman) [inline docs/protocol/Security.md:3]: `retroactively`, not `retractively`

File: docs/protocol/Security.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# Protocol Security
```
Comment (ElliotFriedman) [inline docs/protocol/Security.md:1]: Should add a section here that talks about roles within the system and how we assume that the governor is not acting maliciously, similar to yearn's section on this role https://docs.yearn.finance/security/index#security-assumption

File: docs/protocol/PCVDeposits.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
+# PCV Deposits
+
+The PCV Deposit is a standard interface implemented across the FEI protocol codebase.
+
+A PCV Deposit is a contract that hold funds on behalf of the protocol. There are multiple types (implementations) of PCV Deposits, and each type of PCV Deposit can be instanciated multiple times on-chain. Collectively, all the funds deposited across various PCV Deposits constitute the [Protocol Controlled Value](PCVManagement.md).
+
+For instance, an `ERC20CompoundPCVDeposit` ([link]([aaaa](https://github.com/fei-protocol/fei-protocol-core/blob/develop/contracts/pcv/compound/ERC20CompoundPCVDeposit.sol))) can deposit and withdraw a given `token` to and from a given `cToken` (Compound protocol interest-bearing deposit token). There are multiple on-chain deployments of the `ERC20CompoundPCVDeposit`, such as the DAI Compound PCVDeposit, that deposits DAI on Compound, or the FEI Fuse Pool 8 PCVDeposit, that deposits FEI in Fuse pool 8.
```
Comment (ElliotFriedman) [inline docs/protocol/PCVDeposits.md:7]: `[aaaa]`?

File: contracts/external/WETH9.sol [hunk 4]
```diff
@@ -1,90 +0,0 @@
-/**
```
Comment (ElliotFriedman) [inline contracts/external/WETH9.sol:1]: Removing this contract causes tests to break. Either remove the failing tests as well, or add this contract back.

**Review by ElliotFriedman — CHANGES_REQUESTED** [review]

**Comment by ElliotFriedman** [comment]
Overall, things look really good here, will be great to get these improved docs up!

**Comment by thomas-waite** [comment]
Closing as now in a seperate repo here: https://github.com/fei-protocol/docs and using docusaurus rather than github pages

**Pull request closed without merging**